Tools/machines/tuolumne-llnl/tuolumne_mi300a.flux

```diff
@@ -44,8 +44,8 @@ export MPICH_OFI_NIC_POLICY=GPU
 #export HUGETLB_MORECORE=yes
 
 # threads for OpenMP and threaded compressors per MPI rank
-#   note: 16 avoids hyperthreading (32 virtual cores, 16 physical)
-export OMP_NUM_THREADS=16
+#   note: 21 physical cores per socket maximum (system reserves 3)
+export OMP_NUM_THREADS=21
```

Comment:
We might have kept this to 16 because of the TLS issue (see other comment)
```suggestion
export OMP_NUM_THREADS=16
```

Reply:
No response yet :(

Reply:
No improvement building all-static libs.

Hm, I think the problem is indeed the libsci itself...

Reply:
Ok, unloading libsci solves it.